web/src/styles/global.css

```
@@ -111,12 +111,15 @@
   --blue: var(--cyan-500);
   --blue-bg: var(--cyan-100);
   --font-sans:
-    -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
+    "Geist Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.11.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)